invenio_communities/invitations/services/actions.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
```

Comment:
Minor: can we follow same pattern as already used here: https://github.com/inveniosoftware/invenio-rdm-records/blob/master/invenio_rdm_records/requests/community_submission.py - a request all in one file

Reply:
sure